modules/ffmpeg/producer/ffmpeg_producer.cpp

```diff
@@ -637,7 +652,8 @@ void describe_producer(core::help_sink& sink, const core::help_repository& repo)
 	sink.example(L">> PLAY 1-10 v4l2:///dev/video0", L"to use a web camera as video input on Linux.");
 	sink.para()->text(L"The FFmpeg producer also supports changing some of the settings via ")->code(L"CALL")->text(L":");
 	sink.example(L">> CALL 1-10 LOOP 1");
-	sink.example(L">> CALL 1-10 START 10");
+	sink.example(L">> CALL 1-10 IN 10");
+	sink.example(L">> CALL 1-10 OUT 60");
 	sink.example(L">> CALL 1-10 LENGTH 50");
 	sink.example(L">> CALL 1-10 SEEK 30");
 	core::describe_framerate_producer(sink);
```

Comment:
No problem, I will do it in my original PR #501.
I should have waited, really, with this PR until #501 is merged, as this PR depends on #501. I will tweak my patch and once #501 is in, will rebase this PR on top of 2.1.0 branch.